stats/record.go

```diff
@@ -94,7 +97,7 @@ func Record(ctx context.Context, ms ...Measurement) {
 	if len(ms) == 0 {
 		return
 	}
-	recorder := internal.DefaultRecorder
+	recorder := InternalMeasurementRecorder
```

Comment:
Can you help me double-check my reasoning here?  In theory, this could break someone who was overriding internal.DefaultRecorder, but that isn't possible, since it is in an internal package.

As long as this is backwards-compatible, I think we can accept the change.

Reply:
I agree with your assessment; its backwards compatible because its `internal` and thus cannot be used. However, it does add a new public field that a user could start messing with when they probably shouldn't. I could not find a way to avoid this without introducing an import loop unfortunately.

Reply:
I refactored it to avoid leaking into the public API, and its still backwards compatible since they could not override `internal.DefaultRecorder` as mentioned above.